main_test.go

```diff
@@ -48,7 +48,10 @@ func TestMain(m *testing.M) {
 		}
 	}
 	ag.makeSignerIndex()
-	log.Printf("autographer: %+v\n", ag)
+	// ok this is lame but the contentsignaturepki signer will upload a file dated
+	// at the given second so we don't want anything else to run at the same second
+	// otherwise that file may get rewritten. Easiest way to solve it? Sleep.
+	time.Sleep(time.Second)
```

Comment:
I will hang my head in shame at this one.... if you have any idea how to better handle this, I'm all ears.